src/interfaces/IRedemption.sol

```diff
@@ -21,11 +21,13 @@ interface IRedemption {
     /// @param redeemer The address of the entity redeeming
     /// @param to The receiver address for the redeemed USDC
     /// @param superstateTokenInAmount The amount of SUPERSTATE_TOKEN to redeem
-    /// @param usdcOutAmount The amount of USDC the redeemer gets back
+    /// @param usdcOutAmountAfterFee The amount of USDC the redeemer gets back, after the fee is deducted
+    /// @param usdcOutAmountBeforeFee The amount of USDC the redeemer gets back, before the fee is deducted
     event RedeemV2(
-        address indexed redeemer, address indexed to, uint256 superstateTokenInAmount, uint256 usdcOutAmount
+        address indexed redeemer, address indexed to, uint256 superstateTokenInAmount, uint256 usdcOutAmountAfterFee, uint256 usdcOutAmountBeforeFee
```

Comment:
do we want to create a `RedeemV3` here? cc @jonwalch

Reply:
no, redeemv2 was never delpoyed

Reply:
@jakegsy https://github.com/superstateinc/ustb/blob/main/src/SuperstateToken.sol#L586 can you update the naming here to be consistent with your changes on this pr?